packages/@core.codegen/templates/schema00/domains/{{domain}}/utils/clientSchema/{{name}}.ts

```diff
@@ -38,12 +38,38 @@ function convertToUIFormState (state: I{{ name }}UIState): I{{ name }}FormState
     return result as I{{ name }}FormState
 }
 
-function convertToGQLInput (state: I{{ name }}FormState): {{ name }}UpdateInput {
+function convertToGQLInput (state: I{{ name }}FormState, obj: I{{ name }}UIState, initialState: I{{ name }}FormState): {{ name }}UpdateInput {
```

Comment:
You need to test it on tickets first. When you save a ticket you do not always have all information of initial state. 
1. Changing ticket status
2. Editing ticket without touching existed client